main.c

```diff
@@ -1124,7 +1125,7 @@ main
           mutt_list_insert_tail(&cc_list, mutt_str_dup(optarg));
           break;
         case 'D':
-          dump_variables = true;
+          dump_variables++;
```

Comment:
Not a big fan of this tbh. A separate variable would be clearer.

Reply:
Yeah, done